### PR TITLE
Add ZORA network details

### PR DESCRIPTION
**Pull request by iainnash**
**Description**

The ZORA network information is not included in these token lists.

**Updates**

1. Add Zora
2. Run lint
3. Add ETH details
4. Add testnet USDC Bridged details

**Tests**

The updated data is static and the generate scripts run correctly.

**Additional context**

The only net new other than well-known chain configuration for ZORA is a new deployment of FiatOVMToken to Sepolia.

This PR is cleaner after merging https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/659

### CODE REVIEW

File: bin/cli.ts [hunk 1]
```diff
@@ -32,8 +32,8 @@ program
       fs.writeFileSync(
         validationResultsFilePath,
         `Below are the results from running validation for the token changes. To ` +
-        `re-run the validation locally run: ` +
-        `pnpm validate --datadir ./data --tokens ${options.tokens}\n\n`
+          `re-run the validation locally run: ` +
```
Comment (iainnash) [inline bin/cli.ts:35]: this was added from `pnpm run lint`

File: optimism.tokenlist.json [hunk 2]
```diff
@@ -6142,7 +6183,7 @@
       "logoURI": "https://ethereum-optimism.github.io/data/USDT/logo.png",
       "extensions": {
         "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1",
-        "baseBridgeAddress": "0xD0204B9527C1bA7bD765Fa5CCD9355d38338272b",
+        "baseBridgeAddress": "0x735aDBbE72226BD52e818E7181953f42E3b0FF21",
```
Comment (iainnash) [inline optimism.tokenlist.json:6186]: this must have been changed in the data folder.

File: src/chains.ts [hunk 3]
```diff
@@ -39,7 +39,7 @@ export const NETWORK_DATA: Record<Chain, Network> = {
     id: 34443,
     name: 'Mode',
     provider: new ethers.providers.StaticJsonRpcProvider(
-      'https://mainnet.mode.network',
+      'https://mainnet.mode.network'
```
Comment (iainnash) [inline src/chains.ts:42]: updated from lint

**Comment by philliprossii** [comment]
Zora Network Mainnet

NAME	VALUE
Network	Zora
Chain ID	7777777
Gas Token	ETH
RPC	https://rpc.zora.energy
Websocket	wss://rpc.zora.energy
Block Explorer	https://explorer.zora.energy

File: src/types.ts [hunk 4]
```diff
@@ -31,8 +31,23 @@ export type Chain =
...
+  'optimism-sepolia',
+  'base',
+  'base-goerli',
+  'base-sepolia',
+  'pgn-sepolia',
+  'pgn',
+  'zora',
+  'zora-sepolia',
+  'mode',
+] as const
```
Comment (roninjin10) [inline src/types.ts:50]: There is a merge conflict here

**Review by roninjin10 — CHANGES_REQUESTED** [review]

File: src/types.ts (hunk 4 above)
Comment (iainnash) [reply src/types.ts]: @roninjin10 Fixed the merge conflict.

File: src/chains.ts [hunk 5]
```diff
@@ -97,6 +100,22 @@ export const NETWORK_DATA: Record<Chain, Network> = {
     ),
     layer: 2,
   },
+  zora: {
```
[the remark this answers is not in the document]
Comment (iainnash) [reply src/chains.ts]: Fixed, will take a look.

**Comment by roninjin10** [comment]
Zora needs to be added to validators

```
[ 'BridgedUSDC', 'ETH', 'WETH' ]
 ELIFECYCLE  Command failed with exit code 1.
error: BridgedUSDC: instance.tokens: is not allowed to have the additional property "zora"
error: BridgedUSDC: instance.tokens: is not allowed to have the additional property "zora-sepolia"
error: ETH: instance.tokens: is not allowed to have the additional property "zora-sepolia"
error: ETH: instance.tokens: is not allowed to have the additional property "zora"
error: WETH: instance.tokens: is not allowed to have the additional property "zora"
error: WETH: instance.tokens: is not allowed to have the additional property "zora-sepolia"
error: final token list is invalid
```

**Comment by iainnash** [comment]
@roninjin10 I've updated the validate script RPCs to work in this PR as well.

Output:

```
warning: BridgedUSDC on chain base token 0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA has overridden symbol
warning: BridgedUSDC on chain base token 0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA has overridden name
warning: BridgedUSDC on chain base-goerli token 0x853154e2A5604E5C74a2546E2871Ad44932eB92C has overridden symbol
warning: BridgedUSDC on chain base-goerli token 0x853154e2A5604E5C74a2546E2871Ad44932eB92C has overridden name
warning: BridgedUSDC on chain optimism token 0x7F5c764cBc14f9669B88837ca1490cCa17c31607 has overridden symbol
warning: BridgedUSDC on chain optimism token 0x7F5c764cBc14f9669B88837ca1490cCa17c31607 has overridden name
warning: BridgedUSDC on chain optimism-goerli token 0x7E07E15D2a87A24492740D16f5bdF58c16db0c4E has overridden symbol
warning: BridgedUSDC on chain optimism-goerli token 0x7E07E15D2a87A24492740D16f5bdF58c16db0c4E has overridden name
warning: BridgedUSDC on chain zora token 0xCccCCccc7021b32EBb4e8C08314bD62F7c653EC4 has overridden symbol
warning: BridgedUSDC on chain zora token 0xCccCCccc7021b32EBb4e8C08314bD62F7c653EC4 has overridden name
warning: BridgedUSDC on chain zora-sepolia token 0xccCcCc4532d20C06726866e328F7478Edabf907B has overridden symbol
warning: BridgedUSDC on chain zora-sepolia token 0xccCcCc4532d20C06726866e328F7478Edabf907B has overridden name
warning: ETH on chain ethereum token 0x0000000000000000000000000000000000000000 is a nonstandard token
warning: ETH on chain optimism token 0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000 is a nonstandard token
warning: ETH on chain goerli token 0x0000000000000000000000000000000000000000 is a nonstandard token
warning: ETH on chain sepolia token 0x0000000000000000000000000000000000000000 is a nonstandard token
warning: ETH on chain optimism-goerli token 0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000 is a nonstandard token
warning: ETH on chain optimism-sepolia token 0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000 is a nonstandard token
warning: ETH on chain zora-sepolia token 0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000 is a nonstandard token
warning: ETH on chain zora token 0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000 is a nonstandard token


```

**Comment by iainnash** [comment]
@roninjin10 The PR has been updated and fixed all merge conflicts. The verify script also now runs.

**Comment by roninjin10** [comment]
Still seeing a validation error though it's not obvious why at first glance

**Comment by iainnash** [comment]
@roninjin10 I believe this script is broken in main. I've corrected all errors associated with our tokenlist locally, however, I can attempt to fix this error.

File: optimism.tokenlist.json [hunk 6]
```diff
@@ -700,7 +700,7 @@
       "decimals": 6,
       "logoURI": "https://ethereum-optimism.github.io/data/BridgedUSDC/logo.png",
       "extensions": {
-        "baseBridgeAddress": "0x3154Cf16ccdb4C6d922629664174b904d80F2C35",
+        "baseBridgeAddress": "0x3e2Ea9B92B7E48A52296fD261dc26fd995284631",
```
Comment (hamdiallam) [inline optimism.tokenlist.json:703]: we'll need to fix this. I believe this is Zora's bridge address?

**Comment by iainnash** [comment]
I can add in a zoraBridgeAddress field?

**Comment by hamdiallam** [comment]
@iainnash I see, it looks like we hardcode the bridge address. I think when generating the token list we should dynamically construct this key using the chain name rather than hardcoding it. We have the chain string and create the name appropriately from it

feel free to add `zoraBridgeAddress` for now here and I can followup with a better fix for this longer term

**Comment by iainnash** [comment]
Great, I can do that.


Iain

**Comment by hamdiallam** [comment]
published #689 . Maybe you can try rebasing this PR on top that and regenerating the tokenlist

**Comment by hamdiallam** [comment]
Hey @iainnash , feel free to reopen after rebasing!

**Pull request closed without merging**